src/Data/Foreign/Class.purs

```diff
@@ -50,10 +50,10 @@ instance intIsForeign :: IsForeign Int where
 instance arrayIsForeign :: (IsForeign a) => IsForeign (Array a) where
   read value = readArray value >>= readElements
     where
-    readElements :: forall a. (IsForeign a) => Array Foreign -> F (Array a)
+    readElements :: forall b. (IsForeign b) => Array Foreign -> F (Array b)
     readElements arr = sequence (zipWith readElement (range zero (length arr)) arr)
 
-    readElement :: forall a. (IsForeign a) => Int -> Foreign -> F a
+    readElement :: forall b. (IsForeign b) => Int -> Foreign -> F b
```

Comment:
Oh, good point. I mechanically transformed the type signature and didn't realize what it was doing.